connectors/MicrosoftGraphSecurity/apiProperties.json

```diff
@@ -1,27 +1,76 @@
 {
   "properties": {
-    "capabilities": [
-      "actions"
```

Comment:
Is it not true anymore? Should it be extended to ["actions","triggers"] instead?

Reply:
Updated it to ["actions", "triggers"]